boundary/collect.py

```diff
@@ -183,32 +175,29 @@ def var_decl_start_loc(decl):
                 return base_type.stub.location
             return decl.location
 
-        if not isinstance(decl, gcc.VarDecl):
-            return
-        if isinstance(decl.context, gcc.FunctionDecl):
-            return
-        loc = gcc.get_location()
-        if loc.file != gcc.get_main_input_filename():
-            return
-        properties = {
-            "name": decl.name,
-            "file": os.path.relpath(decl.location.file),
-            "name_loc": (decl.location.line - 1, decl.location.column - 1),
-            "decl_start_line": var_decl_start_loc(decl).line - 1,
-            "decl_end_line": loc.line - 1,
-            "external": decl.external,
-            "public": decl.public,
-            "static": decl.static,
-            "decl_str": None,
-        }
+        for var in gcc.get_variables():
+            decl = var.decl
+            if not decl.location:
```

Comment:
When will this happen?

Reply:
Yes, such as LC ... ， looks like gcc related.